Directory.Build.props

```diff
@@ -114,6 +114,10 @@
     <UseVSTestRunner>true</UseVSTestRunner>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <LibrariesProjectRoot>$([MSBuild]::NormalizeDirectory('$(MSBuildThisFileDirectory)', 'src', 'Libraries'))</LibrariesProjectRoot>
```

Comment:
I didn't find a property that defined the Libraries folder. I figured I'll need it at least once, so I created it here. Was this the right location?

Reply:
Seems reasonable.  I see a few projects that might benefit from using such a property: https://github.com/search?q=repo%3Adotnet%2Fextensions+Libraries+language%3AXML&type=code&l=XML

Reply:
At some point there was a desire to keep MSBuild-related stuff in https://github.com/dotnet/extensions/tree/main/eng/MSBuild, but something's gone wrong :)
This looks similar to the [Arcade's doing](https://github.com/dotnet/arcade/blob/bebd2de1988036b47ecefc7e72b66ce2f403e43d/src/Microsoft.DotNet.Arcade.Sdk/tools/RepoLayout.props#L48) and I would align the naming with the Arcade:
```suggestion
    <!-- Similar to https://github.com/dotnet/arcade/blob/main/src/Microsoft.DotNet.Arcade.Sdk/tools/RepoLayout.props#L48 -->
    <SrcLibrariesDir>$([MSBuild]::NormalizeDirectory('$(MSBuildThisFileDirectory)', 'src', 'Libraries'))</SrcLibrariesDir>
```